recipe/meta.yaml

```
@@ -1,45 +1,52 @@
 {% set name = "azure-mgmt-storage" %}
-{% set version = "21.0.0" %}
-{% set sha256 = "6eb13eeecf89195b2b5f47be0679e3f27888efd7bd2132eec7ebcbce75cb1377" %}
+{% set version = "20.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.zip
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
-  sha256: {{ sha256 }}
+  sha256: 214f3fde8c91e27d53f2e654a28d15003ad3f6f15c8438a8205f0c88a48d9451
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
-    - msrest >=0.7.1
-    - azure-common ~=1.1
-    - azure-mgmt-core >=1.3.2,<2.0.0
-    - typing_extensions >=4.3.0  # TODO: Remove when py38 is minimum supported
+    - python
+    - msrest >=0.6.21
+    - azure-common >=1.1,<1.2
+    - azure-mgmt-core >=1.3.0,<2.0.0
```

```suggestion
    - azure-mgmt-core >=1.3.1,<2.0.0
```
https://inspector.pypi.io/project/azure-mgmt-storage/20.1.0/packages/8c/24/33da34950a637df9bca4bac4b99be81e060613edf22a54589d17e2b7b51a/azure_mgmt_storage-20.1.0-py3-none-any.whl/azure_mgmt_storage-20.1.0.dist-info/METADATA#line.24

---

ah I missed that, thanks!